model_signing/serialization/serialize_by_file.py

```diff
@@ -149,6 +149,7 @@ def __init__(
         self,
         file_hasher_factory: Callable[[pathlib.Path], file.FileHasher],
         max_workers: int | None = None,
+        ignore_paths: list[str] = [],
```

Comment:
Ideally this would be placed in the `serialize` call. This way, a single serialized can be used to serialize multiple models.

Right now, we initialize a serializer with a list of paths to ignore and then when we serialize a model we give the path to the model. These 2 sets of paths should be treated the same.

Reply:
Added it to the serialize function